test/common/testvm.py

```diff
@@ -999,36 +1033,40 @@ def _start_qemu(self, maintain=False, macaddr=None, wait_for_ip=True, memory_mb=
             lease = self._static_lease_from_mac(macaddr)
             if lease:
                 static_domain_name = self.image + "_" + lease['name']
-
-        # domain xml
-        test_domain_desc_original = ""
-        with open(os.path.join(testinfra.TEST_DIR, "common/test-domain.xml"), "r") as dom_desc:
-            test_domain_desc_original = dom_desc.read()
-
-        # add the virtual machine
-        dom = None
         mac_desc = ""
+        cpu_desc = ""
+        domain_type = "qemu"
+        if os.path.exists("/dev/kvm"):
+            domain_type = "kvm"
+            cpu_desc += "<cpu mode='host-passthrough'/>\n"
+            cpu_desc += "<vcpu>%(cpus)d</vcpu>\n" % { 'cpus': cpus or VirtMachine.cpus or 1 }
+        else:
+            print >> sys.stderr, "WARNING: Starting virtual machine with emulation due to missing KVM"
```

Comment:
I used this approach on purpose to avoid the verbosity checks. I believe it should be printed without exception ... at least for now. The reason for that is that testsuite-prepare takes 45 minutes to run in this mode. And testsuite-prepare has verbosity off by default.